controller-server/src/main/java/com/yahoo/vespa/hosted/controller/role/PathGroup.java

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.hosted.controller.role;
+
+import com.yahoo.restapi.Path;
+
+import java.util.EnumSet;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * This declares and groups all known REST API paths in the controller.
+ *
+ * When creating a new API, its paths must be added here and a policy must be declared in {@link Policy}.
+ *
+ * @author mpolden
+ */
+public enum PathGroup {
+
+    /** Paths used for system management by operators */
+    operator("/controller/v1/{*}",
+             "/provision/v2/{*}",
+             "/flags/v1/{*}",
+             "/os/v1/{*}",
+             "/cost/v1/{*}",
+             "/zone/v2/{*}",
+             "/nodes/v2/{*}",
+             "/orchestrator/v1/{*}"),
+
+    /** Paths used when onboarding and creating a new tenants */
+    onboardingUser("/application/v4/user"),
+
+    onboardingTenant("/application/v4/tenant/{ignored}"),
+
+
+    /** Paths used by tenant/application administrators */
+    tenant("/athenz/v1/",
+           "/athenz/v1/domains",
+           "/application/v4/",
+           "/application/v4/athensDomain/",
+           "/application/v4/property/",
+           "/application/v4/tenant/",
+           "/application/v4/tenant-pipeline/",
+           "/application/v4/tenant/{tenant}",
+           "/application/v4/tenant/tenant1/application/",
+           "/application/v4/tenant/{tenant}/application/{application}",
+           "/application/v4/tenant/{tenant}/application/{application}/deploying/{*}",
+           "/application/v4/tenant/{tenant}/application/{application}/instance/{instance}/job/{job}/{*}",
+           "/application/v4/tenant/{tenant}/application/{application}/environment/dev/{*}",
+           "/application/v4/tenant/{tenant}/application/{application}/environment/perf/{*}",
+           "/application/v4/tenant/{tenant}/application/{application}/environment/{environment}/region/{region}/instance/{instance}/global-rotation/override"),
+
+    /**
+     * Paths used for deployments by build service(s). Note that context is ignored in these paths as build service
+     * roles are not granted in specific contexts.
+     */
+    buildService("/zone/v1/{*}",
```

Comment:
This API is used by users to deploy to `dev`. 
A better solution would probably be to expose a deploy API path with environment (`dev` or `perf`) directly under an application, which picks the correct region, but that is some work and not really part of this.